$NetBSD: patch-ad,v 1.5 2008/06/04 15:05:39 dsainty Exp $

The cast breaks compilation on NetBSD, as iconv() expects const char**.

--- src/portable.cpp.orig	2008-01-01 23:41:08.000000000 +1300
+++ src/portable.cpp	2008-06-05 02:06:00.000000000 +1200
@@ -382,7 +382,7 @@
 // avoid a compile error, that is were the CASTNEEDED is for.
 #if ((defined(_LIBICONV_VERSION) && (_LIBICONV_VERSION>=0x0109) && \
       !((defined(_OS_MAC_) || defined(Q_OS_MACX) )&& (_LIBICONV_VERSION==0x010B))) \
-    || defined(_OS_SOLARIS_))
+    || defined(_OS_SOLARIS_) || defined(_OS_NETBSD_))
 #define CASTNEEDED(x) (x)
 #else
 #define CASTNEEDED(x) (char **)(x)
